packages/block-library/src/rss/index.php

```diff
@@ -5,6 +5,46 @@
  * @package WordPress
  */
 
+/**
+ * Helper function to trim text to a certain number of characters while preserving whole words.
+ *
+ * @param string $to_trim The text to trim.
+ * @param int    $character_count The maximum number of characters to return.
+ * @param string $more The string to append to the trimmed text.
+ */
+function trim_to_word_boundary( $to_trim, $character_count, $more = ' [&hellip;]' ) {
```

Comment:
Careful with these new functions: they aren't prefixed, nor namespaced, nor gated behind `function_exists` checks. Function name collisions cause PHP fatals, and this could happen either by colliding with Core or with any third-party plugin.

Please have a look at other core blocks with helper functions to see how they do it. To get a full listing of such helper functions, run:

```sh
find packages/block-library/src/ -name '*.php' | xargs grep -Eoh '^\s*function \w+' | sort | awk '{ print $2 }'
```

Oh, and they should be private functions.

Reply:
thank you, good catch!